src/appengine/handlers/cron/triage.py

```diff
@@ -249,12 +252,62 @@ def _check_and_update_similar_bug(testcase, issue_tracker):
 
   return False
 
+def _get_job_bugs_filing_max(job_type):
+  """Get bugs filing max for given job.."""
+  if job_type in bug_filing_max_24_hours_per_job:
+    return bug_filing_max_24_hours_per_job.get(job_type)
+  
+  max_bugs = None
+  job = data_types.Job.query(data_types.Job.name == job_type).get()
+  if job and 'BUG_FILING_MAX_24_HOURS_PER_JOB' in job.get_environment():
+    max_bugs = int(job.get_environment().get('BUG_FILING_MAX_24_HOURS_PER_JOB'))
```

Comment:
I don't know about convention, but I think catching the error here is probably a good idea since the jobs defintions are handwritten by humans and it would be better not to break triage because someone set `BUG_FILING_MAX_24_HOURS_PER_JOB = 1.0` or `BUG_FILING_MAX_24_HOURS_PER_JOB =q`